product_docs/docs/eprs/7/02_overview/02_replication_concepts_and_definitions/10_sync_replication_log_based/02_logical_replication_slots.mdx

```diff
@@ -38,24 +38,24 @@ When enabled, the logical replication slot is replicated to the standby. If the
 
 ![Logical replication slots with publication database standby after failover](/eprs/latest/images/eprs_logical_replication_failover_600w.svg)
 
-EPRS creates logical replication slot(s) on the publication database primary. If you enabled logical replication slot synchronization while configuring physical replication between publication database primary and publication database standby, then the logical replication slot(s) created on the primary are synchronized to the standby. In case the primary fails, the replication server can continue replication from the standby.
+EPRS creates logical replication slots on the publication database primary. If you enabled logical replication slot synchronization while configuring physical replication between publication database primary and publication database standby, then the logical replication slots created on the primary are synchronized to the standby. If the primary fails, the replication server can continue replication from the standby.
 
 ### Enabling
 
 To enable logical replication slot synchronization between primary and standby publication database servers in a physical streaming replication:
 
--   Add the following line into the `xdb_pubserver.conf` file before setting up a logical replication cluster.
+1. Add the following line into the `xdb_pubserver.conf` file before setting up a logical replication cluster.
```

Comment:
I'm assuming these were meant to be ordered items. And the third item doesn't seem to need a bullet/number since it's an explanation, not an instruction.